com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -803,6 +713,7 @@ else if (path.endsWith(".html")) {
             String email = getOptionsValue(cmd, 'e', null);
             String auth = getOptionsValue(cmd, 'u', null);
             Project project = createProject(classLoader, rootDirectory, buildDirectory, email, auth);
+            EngineArtifactsProvider.setCacheBase(new File(project.getBuildCachePath()));
```

Comment:
We create cache folders for build we get from extender, here we reuse it for vanilla engine as well